src/models/Org.js

```diff
@@ -0,0 +1,19 @@
+const fetch = require('isomorphic-fetch');
+const Repo = require('./Repo');
+
+class Org {
+  constructor(org) {
+    this.org = org;
+    this.login = () => this.org.login;
+    this.id = () => this.org.id;
+    this.url = () => this.org.url;
+    this.fetchRepos = () => fetch(`${this.org.repos_url}?access_token=${process.env.TKN}`)
```

Comment:
@michaelmurray6298 I think `fetchRepos` here should probably be a shared method and therefore exist on the prototype, and not in the constructor. Let me know if you're not quite exactly sure what this means in the context of ES6 `class` syntax.